localisation/touhou_map_l_english.yml

```diff
@@ -3,18 +3,33 @@
  gensokyo_continent: "Gensokyo"
  moon_continent: "Moon"
  ###REGIONS###
- gensokyo_region: "Heart of Gensokyo"
+ Eastern_forest: "East Forest"
+ west_forest_reigion: "West Forest"
+ bamboo_forest_region: "Bamboo Forest"
+ Great_Hakurei_Barrier_reigion: "Great Hakurei Barrier"
  netherworld_region: "Netherworld"
  gensokyan_forest_region: "The Great Forest"
  orbital_region: "Orbit of Gensokyo"
  makai_region: "Makai"
+ kappa_delta: "West Foot of the Mountains"
  sunflower_region: "The Sunflower Fields"
  human_region: "Human Capital"
- foot_of_the_mountain_region: "Foot of the Mountains"
+ moon_island_region: "Orbit of Moon"
+ fairy_reigion: "Fairy Lands"
+ north_foot_of_the_mountain_region: "North Foot of the Mountains"
+ south_foot_of_the_mountain_region: "South Foot of the Mountains"
  youkai_mountain_region: "Youkai Mountain"
- former_hell_region: "Former Hell"
- hell_region: "Hell"
+ west_former_hell_region: "Western Former Hell"
+ east_former_hell_region: "Eastern Former Hell"
+ central_former_hell_region: "Central Former Hell"
+ oni_former_hell_region: "South Former Hell"
+ lower_hell: "Lower Hell"
+  upper_hell: "Lower Hell"
+ sanzu_river: "sanzu river"
+ Animal_realm_reigion : "Animal Realm"
+ Great_border: "Great Borderlands"
  moon_region: "Moon"
+ misty_lake: "Misty Lake"
```

Comment:
Should end with `_region`
Also shouldn't have capitalization for the entry (i.e. should be `great_border_region`, similarly with `Animal_realm_reigion` should be `animal_realm_reigion`)